### PR TITLE
manual deploy

### PR DESCRIPTION
**Pull request by lmmrssa**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+          DBINIT_REPO="$DOCKER_ORG/$DOCKER_REPO_TAG:db-init-$BUILD"
+          #branch="${GITHUB_REF#refs/heads/}"
+          echo "The branched that was pushed is $branch"
+          mkdir -p ~/.ssh
+          chmod 700 ~/.ssh
+          echo "$SERVER_HOST ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBFqXBJPFe+pH3L57o1ildAxHssG4lpkloTcw3Wbs64c7bL8M6hR0rre4ufpCKboVLn4trJqbKOPWtFgBJHsgqXA=" > ~/.ssh/known_hosts
+          ssh root@$SERVER_HOST <<EOF
+          docker pull $PLANET_REPO
+          docker pull $DBINIT_REPO
+          #docker tag $PLANET_REPO "$DOCKER_ORG/$DOCKER_REPO:local"
```
Comment (lmmrssa) [inline .github/workflows/deploy.yml:57]: need to uncomment

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+          #docker tag $DBINIT_REPO "$DOCKER_ORG/$DOCKER_REPO:db-init-local"
```
Comment (lmmrssa) [inline .github/workflows/deploy.yml:58]: need to uncomment

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+          docker images
+          # treehouses service planet restart
```
Comment (lmmrssa) [inline .github/workflows/deploy.yml:60]: need to uncomment

File: .github/workflows/deploy.yml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+          echo "The branched that was pushed is $branch"
+          mkdir -p ~/.ssh
+          chmod 700 ~/.ssh
+          echo "$SERVER_HOST ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBFqXBJPFe+pH3L57o1ildAxHssG4lpkloTcw3Wbs64c7bL8M6hR0rre4ufpCKboVLn4trJqbKOPWtFgBJHsgqXA=" > ~/.ssh/known_hosts
+          ssh root@$SERVER_HOST <<EOF
+          docker pull $PLANET_REPO
+          docker pull $DBINIT_REPO
+          #docker tag $PLANET_REPO "$DOCKER_ORG/$DOCKER_REPO:local"
+          #docker tag $DBINIT_REPO "$DOCKER_ORG/$DOCKER_REPO:db-init-local"
+          docker images
```
Comment (lmmrssa) [inline .github/workflows/deploy.yml:59]: can be removed

File: .github/workflows/deploy.yml [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+            SERVER_HOST="${{ github.event.inputs.hostname }}"
+          fi
+          PLANET_VERSION=$(jq '.version' package.json | sed -e 's/^"//' -e 's/"$//')
+          BUILD="$PLANET_VERSION-$GITHUB_REF_NAME-${GITHUB_SHA::8}"
+          echo "$BUILD"
+          BUILD="0.13.4-deploy-d6b59441"
+          PLANET_REPO="$DOCKER_ORG/$DOCKER_REPO_TAG:$BUILD"
+          DBINIT_REPO="$DOCKER_ORG/$DOCKER_REPO_TAG:db-init-$BUILD"
+          #branch="${GITHUB_REF#refs/heads/}"
+          echo "The branched that was pushed is $branch"
```
Comment (lmmrssa) [inline .github/workflows/deploy.yml:50]: need to remove

File: .github/workflows/deploy.yml [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+        run: |
+          if [[ "${{ github.event.inputs.hostname }}" == "" ]]; then
+            SERVER_HOST="planet.dev.ole.org"  
+          else
+            SERVER_HOST="${{ github.event.inputs.hostname }}"
+          fi
+          PLANET_VERSION=$(jq '.version' package.json | sed -e 's/^"//' -e 's/"$//')
+          BUILD="$PLANET_VERSION-$GITHUB_REF_NAME-${GITHUB_SHA::8}"
+          echo "$BUILD"
+          BUILD="0.13.4-deploy-d6b59441"
```
Comment (lmmrssa) [inline .github/workflows/deploy.yml:46]: need to remove

File: .github/workflows/deploy.yml [hunk 7]
```diff
@@ -0,0 +1,61 @@
+name: Manual Deploy
+
+on:
+  push:
```
Comment (lmmrssa) [inline .github/workflows/deploy.yml:4]: need to remove push trigger

**Review by dogi — APPROVED** [review]

**Comment by lmmrssa** [comment]
For current servers we need to have symlink of `planet.yml` and `planet` folder and also we need to have `volume.yml`
Stop current containers and start new from treehouses services

**Pull request merged**